packages/common/src/initiatives/_internal/InitiativeBusinessRules.ts

```diff
@@ -148,4 +149,10 @@ export const InitiativePermissionPolicies: IPermissionPolicy[] = [
     permission: "hub:initiative:manage",
     dependencies: ["hub:initiative:edit"],
   },
+  // permission to create an association group
+  {
+    permission: "hub:initiative:workspace:associationGroup:create",
+    dependencies: ["hub:initiative:workspace:projects", "hub:group:create"],
+    privileges: ["portal:user:addExternalMembersToGroup"],
```

Comment:
Good questions. From Jay:

_How broadly granted: That privilege is included in the default Publisher role, the one we recommend for using Hub

As for why it's useful — without that privilege, users can only create groups with
membershipAccess: "org". In theory that's not awful for association groups, though_

From that we've decided to update the permission to not rely on the priv, and instead, we'll update the UI to allow different options for `membershipAccess` depending on if the user has the priv. 

So this will be updated to remove the priv 👍